virtuoso.sh

```diff
@@ -54,4 +58,7 @@ then
     echo `date +%Y-%m-%dT%H:%M:%S%:z` > .data_loaded
 fi
 
-exec virtuoso-t +wait +foreground
+# NOTE: put back Virtuoso's server port to 8890 before starting up
+crudini --set virtuoso.ini HTTPServer ServerPort ${VIRT_HTTPServer_ServerPort:-8890}
```

Comment:
The user can also mount his own virtuoso.ini which can overwrite the server port.
This won't catch that. You'll need to find out the port at the top, and then restore it here.

I personally don't think it's worth the effort (people can easily remap the ports using docker), but it's not my call :-D